packages/shared/providers.yaml

```diff
@@ -1814,6 +1814,24 @@ dropbox:
         base_url: https://api.dropboxapi.com
     docs: https://docs.nango.dev/integrations/all/dropbox
 
+emarsys:
+    display_name: Emarsys Core API
+    categories:
+        - marketing
+    auth_mode: WSSE
+    proxy:
+        base_url: https://api.emarsys.net
+        headers:
+            Content-Type: application/json
+        verification:
+            method: GET
+            endpoint: /api/v2/settings
+        retry:
+            at: 'X-RateLimit-Reset'
+    token:
+        expires_in_ms: 300000
```

Comment:
How do a long sync behave when a token expires during execution?

Reply:
For every call, Nango checks `credentials.expires_at` to determine whether the credentials should be refreshed before making the call. This ensures that tokens are always fresh before the actual request to the API.